specification/vision/ImageAnalysis/models.tsp

```diff
@@ -0,0 +1,324 @@
+import "@typespec/http";
+
+namespace ImageAnalysis;
+using TypeSpec.Http;
+
+@doc("Query parameters used for the analyze operator")
+model SharedAnalyzeQuery {
```

Comment:
This should be an `alias` unless we think that, for some reason, customers should reason about SharedAnalyzeQuery as a thing.

Reply:
Done! Now using `alias` instead of `model` to define shared query params